Sage/carbon
# refactor: remove browser util (FE-5042)

### Proposed behaviour

Remove browser util

### Current behaviour

The browser util was probably added to solve no longer existing issue and could be removed.

### Checklist




#### QA


### Additional context



### Testing instructions

Code that was removed was affecting modal based components:
- AdvancedColorPicker
- Alert
- Confirm
- Dialog
- DialogFullscreen
- Sidebar
- Toast

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
